schema.ts

```diff
@@ -0,0 +1,222 @@
+/**
+ * The top-level interface of a custom-elements.json file.
+ *
+ * custom-elements.json documents all the elements in a single npm package,
+ * across all modules within the package. Elements may be exported from multiple
+ * modules with re-exports, but as a rule, elements in this file should be
+ * included once in the "canonical" module that they're exported from.
+ */
+export interface PackageDoc {
+  version: number;
+
+  /**
+   * An array of the custom elements this package contains.
+   */
+  tags: TagDoc[];
+}
+
+export interface TagDoc {
+  /**
+   * The tag name of the web component
+   */
+  name: string;
+
+  /**
+   * Relative path to the js file which registers this web component
+   */
+  path?: string;
```

Comment:
Should we call it 'module' ? like in 'pkg.module'?
Just to avoid people shooting UMDs here...